### PR TITLE
Feature/carousel-FeaturedCards

### PR DESCRIPTION
**Pull request by diegobrunetto**

### CODE REVIEW

File: src/components/FeaturedCardsCarousel/index.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import Carousel, { ResponsiveType } from 'react-multi-carousel';
+
+import 'react-multi-carousel/lib/styles.css';
+
+import { FeaturedCards } from '../../lib/types';
+import FeaturedCard from '../FeaturedCard';
+
+const responsive: ResponsiveType = {
+  superLargeDesktop: {
+    // the naming can be any, depends on you.
```
Comment (giancarlol) [inline src/components/FeaturedCardsCarousel/index.tsx:11]: Missed a comment here. 🧐

File: src/components/FeaturedCardsCarousel/index.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+};
+
+const FeaturedCardsCarousel: React.FC<FeaturedCardsCarouselProps> = ({
+  featuredCards,
+}) => (
+  <Carousel
+    swipeable={true}
+    draggable={true}
+    showDots={false}
+    arrows={false}
```
Comment (giancarlol) [inline src/components/FeaturedCardsCarousel/index.tsx:44]: I would leave the arrows, especially for desktop UX. It's weird to drag cards with the mouse.

File: src/components/FeaturedCardsCarousel/index.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+};
+
+type FeaturedCardsCarouselProps = {
+  featuredCards: FeaturedCards[];
+};
+
+const FeaturedCardsCarousel: React.FC<FeaturedCardsCarouselProps> = ({
+  featuredCards,
+}) => (
+  <Carousel
```
Comment (giancarlol) [inline src/components/FeaturedCardsCarousel/index.tsx:40]: All the `true` properties don't require the word **true**, with the `prop name` alone is enough.

File: src/components/FeaturedCardsCarousel/index.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    showDots={false}
+    arrows={false}
+    responsive={responsive}
+    ssr={true}
+    infinite={true}
+    keyBoardControl={true}
+    customTransition="all .3"
+    transitionDuration={500}
+    containerClass="mt-5 pb-10 carousel-container"
+    itemClass="lg:mr-24 mr-5"
```
Comment (giancarlol) [inline src/components/FeaturedCardsCarousel/index.tsx:52]: I think this `itemClass` is breaking the Carousel animation.

File: src/components/FeaturedCardsCarousel/index.tsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Carousel, { ResponsiveType } from 'react-multi-carousel';
+
+import 'react-multi-carousel/lib/styles.css';
+
+import { FeaturedCards } from '../../lib/types';
+import FeaturedCard from '../FeaturedCard';
+
+const responsive: ResponsiveType = {
```
Comment (giancarlol) [inline src/components/FeaturedCardsCarousel/index.tsx:9]: For the moment and for the amount of FeaturedCards we have, I would leave this as follows:

**superLargeDesktop:** 3 items
**largeDesktop:** 3 items
**desktop:** 2 items
**tablet:** 1 item
**mobile:** 1 item

File: src/components/FeaturedCard/index.tsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+  card: FeaturedCards;
+};
+
+const FeaturedCard: React.FC<FeaturedCardsItemProps> = ({ card }) => {
+  return (
+    <div
+      className={`${styles.card} lg:shadow-2xl shadow-xl lg:m-10 md:m-18 sm:m-5 mt-10 px-5 py-10 flex justify-between `}
+    >
+      <div className="relative">
+        <div
```
Comment (giancarlol) [inline src/components/FeaturedCard/index.tsx:16]: You can close this `div` without children like this: `<div />`. Don't need a closing tag.

**Review by giancarlol — COMMENTED** [review]
I really like this update, the Carousel is nice! 😀 Take a look at the changes I marked to see what you think and we are ready to go.

File: src/components/FeaturedCardsCarousel/index.tsx (hunk 2 above)
Comment (diegobrunetto) [reply src/components/FeaturedCardsCarousel/index.tsx:44]: I agree

Comment (diegobrunetto) [reply src/components/FeaturedCardsCarousel/index.tsx:44]: Is it okay if I just leave it like that for now until we define the final design?

**Comment by diegobrunetto** [comment]
I think that looks better now

File: src/components/FeaturedCardsCarousel/index.tsx (hunk 2 above)
Comment (giancarlol) [reply src/components/FeaturedCardsCarousel/index.tsx:44]: It's okay. Leave it like that so we can merge this.

**Review by giancarlol — COMMENTED** [review]
Nice! I thinks it's better now. One more *picky* thing and we're ready to go. I guess it'll be better for the desktop Carousel UX if you disable the text selection.

To achieve this you only need to add `user-select: none` to the `.card` CSS class.

Test it yourself and let me know what you think ☺

**Comment by diegobrunetto** [comment]
> Nice! I thinks it's better now. One more _picky_ thing and we're ready to go. I guess it'll be better for the desktop Carousel UX if you disable the text selection.
> 
> To achieve this you only need to add `user-select: none` to the `.card` CSS class.
> 
> Test it yourself and let me know what you think

That was great! I didn't even know it was possible lol

**Review by giancarlol — APPROVED** [review]
LGTM! 😁

**Pull request merged**